core/menuitem.ts

```diff
@@ -175,6 +177,25 @@ export class MenuItem {
     this.checked = checked;
   }
 
+  /**
+   * Highlights or unhighlights the component.
+   *
+   * @param highlight Whether to highlight or unhighlight the component.
+   * @internal
+   */
+  setHighlighted(highlight: boolean) {
+    this.highlight = highlight;
+    const el = this.getElement();
```

Comment:
I think I'm missing something, but why is the highlight class the only one you have to add or remove manually?  checked and enabled also set the class during creation but not when the setter is called.

Reply:
Tracking issue is fine

Reply:
Filed https://github.com/google/blockly/issues/8801